### PR TITLE
Removed the common resource bundle from Export Packages for MPHealth

### LINKED ISSUE #17353: Add common bundle for MPHealth 1.0/2.x NLS messages
Currently, the NLS messages for each MP Health version is copied over and duplicated, which shows duplicated messages in https://www.ibm.com/docs/en/was-liberty/core?topic=messages-cwmh, since the messageIDs are duplicated.

Multiple message bundles should not re-use the same messageIDs in more than one messages bundle.

**Solution:**
Place the NLS messages in a common bundle so every version, it uses the common messages bundle. 
Update that common message bundle for future addition of NLS messages.

### PR DESCRIPTION
**Pull request by pgunapal**
fixes #17353
- Removes the NLS resources from Export-Packages from all the bnd files for MP Health and adds it to Private-Packages

### CODE REVIEW

**Comment by pgunapal** [comment]
#spawn.fullfat.buckets=com.ibm.ws.microprofile.health_fat,com.ibm.ws.microprofile.health.2.0_fat,com.ibm.ws.microprofile.health.1.0_fat_tck,com.ibm.ws.microprofile.health.2.0_fat_tck,com.ibm.ws.microprofile.health.2.1_fat_tck,com.ibm.ws.microprofile.health.2.2_fat_tck,io.openliberty.microprofile.health.3.0.internal_fat,io.openliberty.microprofile.health.3.0.internal_fat_tck,io.openliberty.microprofile.health.3.1.internal_fat_tck,io.openliberty.microprofile.health.3.1.internal_fat

**Comment by pgunapal** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.microprofile.health_fat,com.ibm.ws.microprofile.health.2.0_fat,com.ibm.ws.microprofile.health.1.0_fat_tck,com.ibm.ws.microprofile.health.2.0_fat_tck,com.ibm.ws.microprofile.health.2.1_fat_tck,com.ibm.ws.microprofile.health.2.2_fat_tck,io.openliberty.microprofile.health.3.0.internal_fat,io.openliberty.microprofile.health.3.0.internal_fat_tck,io.openliberty.microprofile.health.3.1.internal_fat_tck,io.openliberty.microprofile.health.3.1.internal_fat

**Comment by pgunapal** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.microprofile.health_fat,com.ibm.ws.microprofile.health.2.0_fat,com.ibm.ws.microprofile.health.1.0_fat_tck,com.ibm.ws.microprofile.health.2.0_fat_tck,com.ibm.ws.microprofile.health.2.1_fat_tck,com.ibm.ws.microprofile.health.2.2_fat_tck,io.openliberty.microprofile.health.3.0.internal_fat,io.openliberty.microprofile.health.3.0.internal_fat_tck,io.openliberty.microprofile.health.3.1.internal_fat_tck

File: dev/com.ibm.ws.microprofile.health.2.0/bnd.bnd [hunk 1]
```diff
@@ -38,11 +38,12 @@ Import-Package: \
   *
 
 Export-Package: \
-	com.ibm.ws.microprofile.health.resources,\
+	!com.ibm.ws.microprofile.health.resources,\
```
Comment (jhanders34) [inline dev/com.ibm.ws.microprofile.health.2.0/bnd.bnd]: By marking it private, it won't export.  So this line isn't needed.  Same comment for all of the files.

File: dev/com.ibm.ws.microprofile.health/bnd.bnd [hunk 2]
```diff
@@ -38,7 +38,7 @@ Import-Package: \
   *
 
 Export-Package: \
-    com.ibm.ws.microprofile.health.resources; version="1.0", \
+    !com.ibm.ws.microprofile.health.resources,\
```
Comment (jhanders34) [inline dev/com.ibm.ws.microprofile.health/bnd.bnd]: Should it be added to the Private-Package list?

**Review by jhanders34 — CHANGES_REQUESTED** [review]

File: dev/com.ibm.ws.microprofile.health.2.0/bnd.bnd (hunk 1 above)
Comment (pgunapal) [reply dev/com.ibm.ws.microprofile.health.2.0/bnd.bnd]: Updated.

File: dev/com.ibm.ws.microprofile.health/bnd.bnd (hunk 2 above)
Comment (pgunapal) [reply dev/com.ibm.ws.microprofile.health/bnd.bnd]: The `com.ibm.ws.microprofile.health.resources` package was already in the Private-Package list.

**Comment by pgunapal** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.microprofile.health_fat,com.ibm.ws.microprofile.health.2.0_fat,com.ibm.ws.microprofile.health.1.0_fat_tck,com.ibm.ws.microprofile.health.2.0_fat_tck,com.ibm.ws.microprofile.health.2.1_fat_tck,com.ibm.ws.microprofile.health.2.2_fat_tck,io.openliberty.microprofile.health.3.0.internal_fat,io.openliberty.microprofile.health.3.0.internal_fat_tck,io.openliberty.microprofile.health.3.1.internal_fat_tck

**Review by jhanders34 — APPROVED** [review]

File: dev/com.ibm.ws.microprofile.health/bnd.bnd (hunk 2 above)
Comment (jhanders34) [reply dev/com.ibm.ws.microprofile.health/bnd.bnd]: I even looked twice and didn't see it.  I see it now.  I must be blind today.  :)

**Comment by pgunapal** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.microprofile.health_fat,com.ibm.ws.microprofile.health.2.0_fat,com.ibm.ws.microprofile.health.1.0_fat_tck,com.ibm.ws.microprofile.health.2.0_fat_tck,com.ibm.ws.microprofile.health.2.1_fat_tck,com.ibm.ws.microprofile.health.2.2_fat_tck,io.openliberty.microprofile.health.3.0.internal_fat,io.openliberty.microprofile.health.3.0.internal_fat_tck,io.openliberty.microprofile.health.3.1.internal_fat_tck

**Comment by pgunapal** [comment]
Latest PB passed all the relevant tests, the failing test case is not related to the changes to this PR, it also has an existing defect for it : 284566

**Comment by pgunapal** [comment]
#libby

**Pull request merged**